src/approaches/online_nsrt_learning_approach.py

```diff
@@ -0,0 +1,88 @@
+"""An NSRT learning approach that collects and learns from data online.
+
+Example command:
+    python src/main.py --approach online_nsrt_learning --seed 0 \
+        --env cover \
+        --explorer random_options \
+        --max_initial_demos 1 \
+        --num_train_tasks 1000 \
```

Comment:
is this just a dummy large value, since the number of interaction cycles governs the speed of the pipeline?

Reply:
yeah